blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jorge Vicente Cantero and Martin Duhem
+title: "Meet bloop, a fast tool to compile and test your project"
+---
+
+Martin and I are excited to introduce `bloop`, a new tool that we have been working on at the Scala
+Center during the past three weeks.
+
+## What is bloop?
+
+Bloop is a command-line tool for fast edit/compile/test workflows. Its primary goal is to compile
+and test your project as fast as possible, offering a snappy developer experience.
+
+## Why `bloop`?
+
+Build tools are complex applications that need to cover a wide range of use cases, from building and
+testing your project to deploying it, integrating with external tools, resolving dependencies and
+any build-related tasks.
+
+Such a vast array of requirements make build tools like `sbt`, `Maven` and `Gradle` create their own
+configuration models and DSLs to be customizable, whether that is programmatically or declaratively.
+It is easy that in the process of creating a build tool, performance is sacrificed by ease of
+customization, readability or maintenance.
+
+On the other hand, there is also the problem that build tools move slowly and
+have a hard time upgrading to new changes upstream (for example, in compiler
+APIs). For example, `Gradle`, `Maven`, `Leiningen`, and `CBT` have not yet been
+migrated to Zinc 1.0, [which offers faster incremental compilations than its
+predecessor](https://www.scala-lang.org/blog/2017/11/03/zinc-blog-1.0.html).
+As a result, users of these tools are not benefiting from the latest state of
+the art, and that's hurting their developer productivity.
+
+`bloop` aims to fix both problems. It only supports the most common build tasks (`compile`, `test`
+and `console`), and allows other build tools to integrate with it.
```

Comment:
You should mention that `bloop` is not a brand new build tool. Because so far it seems it is! Make it clear that it builds on already-functional sbt builds, and that it's like a command-line version of zinc.